src/utils/galleries.ts

```diff
@@ -0,0 +1,74 @@
+import config from '@config';
+import { hasContentCollection } from '@root/src/content.config';
+
+const BASE_URL = import.meta.env.PUBLIC_BASE_URL;
+
+/**
+ * Fetches data from a URL and parses it as JSON.
+ * @param url
+ */
+export const fetchJson = async (url: string) => {
+  const response = await fetch(url);
+  return await response.json();
+};
+
+/**
+ * The project_ids param disappears from the manifest ID when accessed directly from Core Data,
+ * so we need to truncate it to match the ID in the content collection.
+ * @param manifestId
+ */
+export const truncateManifestId = (manifestId: string) => manifestId.split('?project_ids')[0];
```

Comment:
Could you explain a little more about what we're doing here? My hope was that the `gallery` config option could be used for any manifest, not just ones coming from Core Data.